Scoop/Supporting/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not 100% on this, but I think you shouldn't be pushing this GoogleService-Info file and the Keys one. Can someone else confirm?

Reply:
Yes we should not be pushing this file and should fix this. To fix this, confirm that the GitIgnore contains this file in the right location. Then you want to update the GitIgnore and push a change with that. Finally, to prevent this file from being viewed in the future, you would to squash all your commits into one. Let me know if you need help doing this Richie. (Also don't worry, I did the exact same thing my second semester on AppDev while working on CourseGrab)